utreexo/src/update.rs

```diff
@@ -0,0 +1,45 @@
+use super::hash::Hash;
+use super::proof::{Proof, ProofStep};
+use super::utreexo::Utreexo;
+
+use std::collections::HashMap;
+
+pub struct Update {
+    pub(crate) u: Utreexo,
```

Comment:
nitpick: could we have more spelled out names, because `u.u.` and `hh` in the code are kinda hard to read :-)